kolibri/core/tasks/api.py

```diff
@@ -195,7 +198,7 @@ def create(self, request):
             object, including args, kwargs, and extra_metadata.
         """
         validated_data = self.validate_create_req_data(request)
-
+        print("HI")
```

Comment:
Cleanup needed here!

Reply:
With the print statements, i thought pre-commit hooks shall warn me(like es-lint), ig that is not the case

Reply:
Seems like we don't! There is a flake8 plugin for this, which we could use: https://pypi.org/project/flake8-print/ if you would like to open an issue to add that!

Reply:
sure, will do that